### ISSUE TITLE
Text says image styles available, but they are not available

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/fanciblock#6]
The image field on an image box says that you will be able to select an image style, but there is no option to select an image style. 

admin/config/media/fanciblocks/fanciblock_title/<#boxid>/edit

Need to either change text or allow selection of image style. 

My preference would be to add an option for image styles. 

![Cantalope___Demo](https://user-images.githubusercontent.com/3144571/66448155-46d42880-ea16-11e9-92b9-c3316f281e25.png)

**Comment by stpaultim** [comment]
I would love to get help fixing this.

**Comment by docwilmot** [comment]
i dont think it would make sense to select image styles on a per image basis, and the block config already allows a choice of sizes (applied to all images in a box), so best change the text, no?

**Comment by stpaultim** [comment]
Makes sense.

**Comment by stpaultim** [comment]
@docwilmot - PR provided.
